community/modules/compute/gke-job-template/variables.tf

```diff
@@ -0,0 +1,102 @@
+/**
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+variable "name" {
+  description = "The name of the job."
+  type        = string
+  default     = "my-job"
+}
+
+variable "node_count" {
+  description = "How many nodes the job should run in parallel."
+  type        = number
+  default     = 1
+}
+
+variable "command" {
+  description = "A list of strings that will be joined to create the job command."
+  type        = list(string)
+  default     = ["hostname"]
+}
+
+variable "image" {
+  description = "The container image the job should use."
+  type        = string
+  default     = "debian"
+}
+
+variable "node_pool_name" {
+  description = "The name of the node pool on which to run the job. Can be populated via `use` feild."
```

Comment:
What happens if left null? I think it would be worth specifying.

Reply:
This was converted to a list and default is now empty list.